scipy/stats/tests/test_distributions.py

```diff
@@ -6870,6 +6870,22 @@ def test_gengamma_edge():
     p = stats.gengamma.pdf(0, 1, 1)
     assert_equal(p, 1.0)
 
+@pytest.mark.parametrize("a, ref",
+                         [(1500000.0, -5.621230523748292),
+                         (10000000000.0, -10.05055748355847),
+                         (1e+30, -33.10536137897591),
+                         (1e+60, -67.65137601525164),
+                         (1e+100, -113.70597317167858)])
```

Comment:
As in #18122. instead of five tests with very high values let us also add values for low `c`.

Reply:
The linked issue was only for large values. If I add tests for low `a`, it'd have to be in a different PR.